package.json

```diff
@@ -34,7 +34,7 @@
   "author": "GolemFactory <contact@golem.network>",
   "license": "GPL-3.0",
   "engines": {
-    "node": ">=16.0.0"
+    "node": ">=18.0.0"
```

Comment:
That's a BC breaking change, so please include this in the commit message for semantic release. I guess you want to bump the version of @golem-sdk/cli in a different PR or this one?